src/Engine/AlgoliaEngine.php

```diff
@@ -56,7 +56,7 @@ public function delete($indexName)
         return true;
     }
 
-    public function search($query, $indexName, $page = 1, $nbResults = null, array $parameters = [])
+    public function search($query, $indexName, $page = 1, $nbResults = 20, array $parameters = [])
```

Comment:
I think I didn't want to add an arbitrary value here, to let the user set `hitsPerPage`in the Algolia index settings.
If we have a default value, the default settings will always be overridden by this value.